### PR TITLE
Serializable `AccountInfo`

### PR DESCRIPTION
**Pull request by hardsetting**
Added strongly typed and serializable `AccountInfo`.
This should take care of the problem of serializing public keys.

### CODE REVIEW

File: src/AccountInfo.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  Serializer, SigningScheme
+} from '@aptos-labs/ts-sdk'
+
+export interface AccountInfoInput {
+  address: AccountAddressInput
+  publicKey: PublicKey
+  ansName?: string
+}
+
+export class AccountInfo extends Serializable {
```
Comment (0xmaayan) [inline src/AccountInfo.ts:17]: what does the wallet return the dapp here? the `AccountInfo` class? I really wish we could have the de/serialization logic live in our sdk

Comment (hardsetting) [reply src/AccountInfo.ts:17]: yes, we can return an AccountInfo instance

Comment (hardsetting) [reply src/AccountInfo.ts:17]: do you think this class should live in the sdk? why?

Comment (0xmaayan) [reply src/AccountInfo.ts:17]: dont know if this class but the de\serialization logic

Comment (hardsetting) [reply src/AccountInfo.ts:17]: what are you proposing?

**Pull request merged**